ext/project_data.yaml

```diff
@@ -21,7 +21,6 @@ gem_runtime_dependencies:
   hiera: ['>= 3.2.1', '< 4']
   # PUP-7115 - return to a gem dependency in Puppet 5
```

Comment:
Is this something that can happen now?

Reply:
Sadly not yet. Removing the vendored version, breaks puppetserver. We'll need to do a phased rollout where we add semantic_puppet to puppetserver packaging, then unvendor from puppet and ensure newer puppet-agent packages conflict with older puppetserver packages... Also there are differences between our vendored semantic puppet and the gem that need to be resolved.